clover-conversion/Clover-config.md

```diff
@@ -463,7 +466,7 @@ The following boot-arg should handle 99% of cases(pair this with RTCMemoryFixup)
 rtcfx_exclude=00-FF
 ```
 
-If this works, slowly shorten the excluded area until you find the part macOS is getting fussy on
+If this works, slowly shorten the excluded area until you find the part macOS is getting fussy on. More details available [here](https://dortania.github.io/OpenCore-Post-Install/misc/rtc.html)
```

Comment:
Despite end-user should at least have finished elementary school, hence the ability of reading, adding a link to explain how to proceed in case of RTC (CMOS) errors aren't solved, is better